dask_expr/tests/test_collection.py

```diff
@@ -242,6 +242,15 @@ def test_std_kwargs(axis, skipna, ddof):
     )
 
 
+@pytest.mark.parametrize("func", ["cumsum", "cumprod", "cummin", "cummax"])
+def test_cumulative_methods(df, pdf, func):
+    assert_eq(getattr(df, func)(), getattr(pdf, func)(), check_dtype=False)
+    assert_eq(getattr(df.x, func)(), getattr(pdf.x, func)())
+
+    q = getattr(df, func)()["x"]
+    assert q.simplify()._name == getattr(df.x, func)()
+
+
```

Comment:
Could you add a test for skipna=False (running on data that does include NA's)?

Reply:
added